Sources/PackageLoading/PkgConfig.swift

```diff
@@ -401,6 +397,14 @@ internal struct PCFileFinder {
         }
     }
 
+    public init(brewPrefix: AbsolutePath?) {
+        self.init(pkgConfigPath: brewPrefix?.pathString ?? "pkg-config")
```

Comment:
did we loose the logic in L390 (` brewPrefix.appending(components: "bin", "pkg-config").pathString`)?